clacheck/clacheck.py

```diff
@@ -20,6 +20,8 @@
 FAILURE = 'failure'
 CLAFILE = "/var/cache/openssl/checkouts/omc/cladb.txt" #<EDIT>
 
+CLA_LABEL='hold: cla required'
```

Comment:
Nit: perhaps do like in the rest of the file, have a little bit of spacing around the assignment operator?

Reply:
Done in ada6caa4888b12ae3aa10b97e8343a3f832df291.